shell-and-git/shell-and-git.md

```diff
@@ -0,0 +1,19 @@
+# Helpful Comand Collection
+
+### Important "branch" commands
+
+| command                      | functionality                        |
+| ---------------------------- | ------------------------------------ |
+| `git switch -c <branchname>` | create a new branch and switch to it |
+| `git switch <branchname>`    | switch branches                      |
+| `git branch`                 | list your branches                   |
+| `git branch -a`              | list all branches (local and remote) |
+| `git branch -d <branchname>` | delete a branch                      |
+
+### Some (maybe) usefull hints 💡
+
+- 'git pull' sollte immer auf dem main ausgeführt werden
```

Comment:
Dies gilt nicht nur für den main branch. Grundsätzlich kann man mit git pull alle Änderungen herunterladen, die remote in einem GitHub-Repository vorgenommen wurden, auch andere branches könnten Änderungen haben, die du lokal noch nicht hast.

Reply:
Alles klar. Wird abgeändert. Danke :)